contracts/clients/src/BlsWalletWrapper.ts

```diff
@@ -16,19 +16,39 @@ import {
   BLSWallet__factory,
   // eslint-disable-next-line camelcase
   TransparentUpgradeableProxy__factory,
+  VerificationGateway,
   // eslint-disable-next-line camelcase
   VerificationGateway__factory,
-} from "../typechain";
+} from "../../typechain";
```

Comment:
This is reaching out to `contracts/typechain` which is incorrect. `contracts/clients` needs to be self-contained so that it can be published on npm. `contracts/clients/typechain` in the repo is a symlink to `contracts/typechain`.

Reply:
Addressed in https://github.com/web3well/bls-wallet/pull/351